website/frontend/src/pages/Press/Press.js

```diff
@@ -4,83 +4,106 @@ import TheConversationIcon from 'icons/press/the-conversation.svg';
 import ZindiIcon from 'icons/press/zindi.svg';
 import DevexIcon from 'icons/press/devex.svg';
 import PMLDailyIcon from 'icons/press/pml-daily.svg';
+import LifeWireIcon from 'icons/press/Lifewire-logo.svg';
```

Comment:
Thanks @FaithDaka for this, it is awesome that we have our imaged files appropriately stored in their own directory.  :)

I was also wondering, could we apply some level of consistency with our image file naming conventions?

Reply:
Sure. There can be a workshop for that

Reply:
Alright @FaithDaka , perhaps to be more direct.....shouldn't this be be `LifeWireLogo.svg` or `livewire.svg`? Based on what I am seeing elsewhere?

We could have this implemented in a future PR or just attempt to address the thought within this very PR?

**Context:** this might help us to minimise the re-occurance of the Google.svg issue we witnessed sometime back.

Happy to learn about your additional thoughts.

Reply:
The Google.svg error was due to **casing** insensitivity of certain operating systems rather than **naming** of the file as far as I can recall. We prefer to use the actual names (in whatever casing) for naming such that web-crawlers easily correlate a named image to a search result. That's why you'll find some search results have more accurately returned images than others. 
So LifeWireLogo.svg and lifewirelogo.svg will still cause a casing issue if placed in the same folder @Baalmart

Reply:
Alright, thanks for the clarifications @FaithDaka .
I will also leave a JIRA task for next sprint around your initial "workshop" suggestion.